dashboards/ec2/docs/ec2_application_load_balancer_detail.md

```diff
@@ -0,0 +1,3 @@
+This dashboard answers the following questions:
+
+- What relationship does this load balancer have with other resources?
```

Comment:
```suggestion
- What relationships does this load balancer have with other resources?
```

The word `relationships` should be plural, can you please fix this here and all other instances of it?